plugins/roadie-backstage-entity-validator/package.json

```diff
@@ -48,7 +48,7 @@
     "minimist": "^1.2.5"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.33.1",
+    "@backstage/cli": "backstage:^",
```

Comment:
These should be manually versioned and not use backstage:^. Can you use explicit versions here please?

Reply:
Thanks for the catch, it should be fixed now 🙏🏻